### ISSUE TITLE
[Accessibility] Screen readers like #HashtagsLikeThis, but the Mastodon UI only offers #hashtagslikethis

### ISSUE THREAD

**Issue by Cassolotl** [tootsuite/mastodon#8241]
Screenshot:

![screen shot 2018-08-17 at 11 43 06](https://user-images.githubusercontent.com/6607512/44262403-d0bda280-a212-11e8-8e3a-ab833ddb0e6a.png)

* * * *

- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.

**Comment by contraexemplo** [comment]
+1. It's incredibly annoying to write hashtags right only to be corrected by Mastodon's UI.

**Comment by Cassolotl** [comment]
"""corrected""" :D

**Comment by mal0ki** [comment]
This has been bothering me for a while, yeah.

**Comment by ThibG** [comment]
Yeah, that has been bothering me as well. Not sure how we could handle it, though. All hashtags stored so far are lowercased. We can't store every capitalization either. We could maybe have the suggestion copy the capitalization for the substring that has already been typed, but it's not really a good fix either.

**Comment by Cassolotl** [comment]
It sounds like if people want to make a post that is accessible to screen readers they have to do it the long way, taking more effort and risking misspellings and stuff. :/

**Comment by Gargron** [comment]
Well, the obvious solution is to change the tags index to case-insensitive instead of lowercasing before saving. Then, whatever casing the first use of a hashtag is, it will keep that for all others.

On the other hand, I don't think the autosuggest in the OP comes from the database at all, actually. It's localStorage and there is no reason for that to store lowercased tags, you could just store them the way the user typed them before. I think that might be the main bug here.
